samples/drivers/adc/boards/lpcxpresso55s36.overlay

```diff
@@ -26,7 +26,7 @@
 		reg = <0>;
 		zephyr,gain = "ADC_GAIN_1";
 		zephyr,reference = "ADC_REF_EXTERNAL0";
-		zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+		zephyr,acquisition-time = <ADC_ACQ_TIME(ADC_ACQ_TIME_TICKS, 7)>;
```

Comment:
is there a reason for 7 in particular?

Reply:
For the ADC, the longer the sampling time is set, the higher the accuracy of the data obtained, but too high a sampling time will cause the sampling rate to decrease, so I chose a more compromised configuration option.